site/gatsby-site/src/utils/checklists.js

```diff
@@ -0,0 +1,121 @@
+import { classy } from 'utils/classy';
+import { faShield, faWarning } from '@fortawesome/free-solid-svg-icons';
+
+const abbreviatedTag = (tag) => tag.replace(/^.*:/g, '');
+
+const risksEqual = (risk1, risk2) => {
+  if (risk1.generated != risk2.generated) {
+    return false;
+  }
+  if (risk1.generated && risk2.generated) {
+    return risk1.title == risk2.title;
+  }
+  if (!risk1.generated && !risk1.generated) {
+    return risk1.id == risk2.id;
+  }
+};
+
+const Label = classy('label', 'mb-1 block');
+
+const emptyRisk = () => ({
+  id: Math.random().toString(36).slice(-10),
```

Comment:
This doesn't look sound to me. How are we generating client-side UUID's currently?

Reply:
I grepped for `uuid` and found that we were using `uuidv4` elsewhere, so I switched to that. I'm unsure whether it would be better to use sequential numeric IDs instead, or maybe even something cute like a random sequence of words.

Reply:
I think using `uuidv4` is good for now. There's a sense to having incremental (almost human-readable) IDs but I'd rather not give out information like that unless there's a case for the user.

We can also think about this further when deciding how checklists will be published or shared, not just identified within the DB rows.